coldfront/core/allocation/views.py

```diff
@@ -257,7 +258,11 @@ def post(self, request, *args, **kwargs):
                     allocation_remove_user.send(
                         sender=self.__class__, allocation_user_pk=allocation_user.pk)
 
-                send_allocation_customer_email(allocation_obj, 'Allocation Denied', 'email/allocation_denied.txt', domain_url=get_domain_url(self.request))
+                if new_status == 'Denied':
+                    messages.warning(request, "Denying an allocation that already exists. Did you mean to revoke the allocation instead?")
```

Comment:
Curious why are we issuing this message? Is this case handling the user "Denying" and allocation that is already in "Denied" status?

Reply:
I do like this idea but it opens up a can of worms because we don't do this in other places.  I think for now, let's remove it to keep it consistent with other status work flow.

Reply:
Sounds good. @rg663 this PR looks good, if you can just remove that message we can get this merged. Thanks!